src/backend/joanie/core/serializers/client.py

```diff
@@ -393,7 +393,7 @@ class Meta:
         read_only_fields = fields
 
 
-class ContractSerializer(serializers.ModelSerializer):
+class ContractSerializer(AbilitiesModelSerializer):
```

Comment:
In this case, I think you should set `exclude_abilities=True` into `OrderSerializer`